src/Azure.Functions.Cli/Actions/HostActions/StartHostAction.cs

```diff
@@ -831,5 +841,9 @@ private void EnsureWorkerRuntimeIsSet()
             // Update local.settings.json
             WorkerRuntimeLanguageHelper.SetWorkerRuntime(_secretsManager, GlobalCoreToolsSettings.CurrentWorkerRuntime.ToString());
         }
+        private void PrintMigrationWarningForDotnet6Inproc() 
```

Comment:
empty line between methods
```suggestion
       
         private void PrintMigrationWarningForDotnet6Inproc() 
```